packages/website/src/components/Playground.module.css

```diff
@@ -7,30 +7,34 @@
   width: 20rem;
   background: var(--playground-main-color);
   overflow: auto;
+  z-index: 1;
 }
 
 .sourceCode {
   height: 100%;
-  width: 50%;
   border: 1px solid var(--playground-secondary-color);
 }
 
 .codeBlocks {
   display: flex;
+  flex-grow: 1;
+  flex-shrink: 1;
+  flex-basis: 0;
   flex-direction: row;
   height: 100%;
   width: calc(100vw - 20rem);
 }
 
 .astViewer {
   height: 100%;
-  width: 50%;
+  width: 100%;
   border: 1px solid var(--playground-secondary-color);
   padding: 0;
   overflow: auto;
   word-wrap: initial;
   white-space: nowrap;
   background: var(--code-editor-bg);
+  z-index: var(--ifm-z-index-fixed);
```

Comment:
Why add this? I tried removing it locally and didn't see anything bad as a result.

Reply:
Good point, I added that in the vanilla DOM-based implementation and forgot to remove after installing `react-split-pane`. Removed now.